UnityProject/Assets/Scripts/ChemistryComponents/ReagentContainer.TransferInteraction.cs

```diff
@@ -27,6 +27,8 @@ public partial class ReagentContainer : ICheckedInteractable<HandApply>, //Trans
 	{
 		[Header("Transfer settings")]
 
+		[SerializeField] private bool isInteractable = true;
```

Comment:
I don't think you need at this as an option to make it not intractable with Reagent containers already?

Reply:
![Uploading image.png…]()

Reply:
No transfer might be doing other stuff in the background, this is why we disable interactions completely on `WillInteract()` rather than using that enum in wherever it is being used.

Reply:
that's an odd answer. The transfer mode is what you need to change to avoid transfering reagents into the splats. That's the issue you are trying to fix.

Reply:
yet the no transfer mode doesn't work, but my fix does; curious.

Reply:
if it doesn't work, maybe see why it doesn't? Instead of adding another bool to the component.

Reply:
the closest example for how an interaction rework should look like is actually in this same reagent system as well; take a look at `TransferResult` and see how we can apply this bad boy to interactions.
It would give us finer control over things.

Reply:
well, regardless of all you said, we can't just make them non-interactable because we sometimes want to interact with them. Grabbing samples from blood splats or vomit and analyse their content. Grabbing sample from a whatever chemical splat to see what is there.

Reply:
the issue is not that they are interactable but that the container is allowed to receive transfer when it should be an out only operation.

Reply:
> we can't just make them non-interactable because we sometimes want to interact with them. Grabbing samples from blood splats or vomit and analyse their content. 

![image](https://user-images.githubusercontent.com/34368774/232160501-0f9f4ee0-8059-4dcc-a9e9-fdcb30f82fc7.png)

...good point.
I'll revise my solution.